OpenSim/Simulation/Model/ControllerSet.h

```diff
@@ -86,7 +70,7 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ControllerSet, ModelComponentSet<Controller>);
 
 private:
 
-    std::unique_ptr<Storage> _controlStore;
+    SimTK::ClonePtr<Storage> _controlStore;
```

Comment:
> If someone makes a copy of the model, should the control storage stay around? I'm not sure but I think not.

I agree in principle, but the _controlStorage is a result of the model's controllers, and not an input like other storages  like input kinematics and ground reaction forces. If for some reason you cloned the model before asking to write results, you would lose all the controls computed during a simulation (e.g. CMC, RRA). Since the storage is reset when you rerun a simulation, I didn't see the harm in keeping it around.  This could be a separate issues as well.